percona_playback/query_log/query_log.cc

```diff
@@ -137,13 +141,16 @@ void* ParseQueryLogFunc::operator() (void*)  {
       tmp_entry.reset(new QueryLogEntry());
       (*this->nr_entries)++;
     }
-
+    
```

Comment:
You are right ser. It is fault of IDE and my careless. I removed all trailing spaces which you commented. You can see on the commit with title "Update query_log.cc 15.11.2016"